summarizer/summarizer.py

```
@@ -0,0 +1,434 @@
+
+    @commands.Cog.listener()
+    async def on_message_without_command(self, message: discord.Message):
+        if message.author.bot:
+            return
+
+        user_data = await self.config.user(message.author).all()
+        if user_data.get("is_afk"):
+            await self.config.user(message.author).is_afk.set(False)
+            embed = discord.Embed(
+                title="Welcome back",
+                description=f":sparkles: Generating AI summary",
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)